acquia_lift.module

```diff
@@ -65,6 +65,10 @@ function acquia_lift_page_attachments(array &$page) {
   $page_context = \Drupal::service('acquia_lift.service.context.page_context');
   $page_context->setPageContextTitle($title);
   $drupal_settings['pageContext'] = $page_context->getAll();
+  $metatags = $page_context->getMetatags();
+  $metatags_lift = $page_context->getLiftMetatags($credential_settings);
```

Comment:
We can merge getLiftMetatags() into getMetatags(), and just call it getMetatags(). I don't think there is a need for two separate calls.